cmd/vpc/ethlink/destroy/public.go

```diff
@@ -15,14 +15,14 @@ import (
 )
 
 const (
-	_CmdName      = "destroy"
-	_KeyEthLinkID = config.KeyEthLinkDestroyID
+	cmdName      = "destroy"
```

Comment:
Yeah, I think this is a step backward for readability, but I don't care enough to do anything beyond pointing it out and moving on.